src/nanoemoji/config.py

```diff
@@ -203,6 +214,12 @@ def load(config_file: Path = None, additional_srcs: Tuple[Path] = None) -> FontC
     output_file = _pop_flag(config, "output_file")
     color_format = _pop_flag(config, "color_format")
     upem = int(_pop_flag(config, "upem"))
+    ascent = int(_pop_flag(config, "ascent"))
+    descent = int(_pop_flag(config, "descent"))
+    transform = _pop_flag(config, "transform")
+    if not isinstance(transform, Affine2D):
```

Comment:
isn't this flag value always going to be a string? the `if` may be unnecessary

Reply:
the default Config.transform is Affine2D.identity